ext/reflection/php_reflection.c

```diff
@@ -1882,7 +1882,7 @@ ZEND_METHOD(ReflectionFunctionAbstract, getAttributes)
 
 	GET_REFLECTION_OBJECT_PTR(fptr);
 
-	if (fptr->common.scope && !(fptr->common.fn_flags & ZEND_ACC_CLOSURE)) {
+	if (fptr->common.scope && ((!(fptr->common.fn_flags & ZEND_ACC_CLOSURE) || fptr->common.fn_flags & ZEND_ACC_FAKE_CLOSURE))) {
```

Comment:
You have a redundant pair of parentheses here, did you rather mean

```suggestion
	if (fptr->common.scope && (!(fptr->common.fn_flags & ZEND_ACC_CLOSURE) || (fptr->common.fn_flags & ZEND_ACC_FAKE_CLOSURE))) {
```

? (ah I also see that bwoebi suggested a "factorized" version)

Reply:
Yeah you're right, the current parentheses don't make sense. But they'll become redundant with Bobs suggestion anyway.